src/Integration/Helpers/SonarLintOutputLogger.cs

```diff
@@ -49,5 +46,14 @@ public void WriteLine(string messageFormat, params object[] args)
         {
             VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, messageFormat, args);
         }
+
+        public void LogDebug(string messageFormat, params object[] args)
```

Comment:
Tests are added in https://github.com/SonarSource/sonarlint-visualstudio/pull/3439